pages/partner.jsx

```diff
@@ -39,6 +43,33 @@ export default function PartnerForm() {
   return (
     <Layout>
       <Container>
+      <CardRowContainer>
+
+<CardOne
+  title="Make a Donation"
+  description="A tax-deductible donation below helps us provide our Partners with the tools they need."
+  icon={<AccessibilityIcon fontSize="large" sx={{ color: "white" }} />}
+  buttonText="Donate"
+  buttonLink="#"
+/>
+
+<CardOne
+  title="Sponsor Us"
+  description="Help our Partners increase their nonprofit’s impact by becoming an ally and offering support."
+  icon={<AccessibilityIcon fontSize="large" sx={{ color: "white" }} />}
+  buttonText="Sponsor"
+  buttonLink="#"
+/>
+
+<CardOne
+  title="Share Our Mission"
+  description="Spread the word with your friends, coworkers, and social media, to help continue our mission."
+  icon={<AccessibilityIcon fontSize="large" sx={{ color: "white" }} />}
+  buttonText="Share"
+  buttonLink="#"
+/>
+
+</CardRowContainer>
```

Comment:
It looks like these were accidentally left here.

Reply:
Yep I was just testing out stuff-- will remove both chunks of code!